bench.test.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+import subprocess
+from bench import main
+
+class TestBench(unittest.TestCase):
```

Comment:
This file has some initial unit tests for argument parsing.  I don't plan to extend it further for this PR.